src/jdk.jshell/share/classes/jdk/internal/jshell/tool/resources/l10n.properties

```diff
@@ -182,7 +182,7 @@ jshell.console.completion.all.completions.number = <press tab again to see all p
 jshell.console.completion.all.completions = <press tab again to see all possible completions>
 jshell.console.no.javadoc = <no documentation found>
 jshell.console.do.nothing = Do nothing
-jshell.console.choice = Choice: \
+jshell.console.choice = Choice:\u0020
```

Comment:
I see this backslash is also being removed. It does seem misplaced since none of the other lines use it. But just double checking with you that this is the desired change because for the other locales, the change was to just translate the space to unicode.

Reply:
Thanks for reviewing Damon, the `\` is used to signify the white space after the `:` is intentional. However, using a trailing `\u0020` is more intentional and clear, and allows for consistency between the original and localized versions, so I manually made this change.

Reply:
Sure, I agree. Just checking to see if this was a product of the translation tool used, because if so, this might've been a mistake or issue with the tool. Looks fine here then